src/test/ui/consts/const-eval/const-eval-intrinsic-promotion.rs

```diff
@@ -0,0 +1,5 @@
+#![feature(core_intrinsics)]
+fn main() {
+    let x: &'static usize =
```

Comment:
Please add a comment along the lines of "ensure that intrinsics do not get implicitly promoted".

Isn't there an existing test that we could reasonably add this to?

Reply:
There was no test for this case. This was the test that I was running locally to determine the application of whitelist.